moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -119,6 +122,9 @@ def to_dict(self):
         parameters.update(self.planning_pipelines)
         parameters.update(self.trajectory_execution)
         parameters.update(self.planning_scene_monitor)
+        if self.sensors_3d:
```

Comment:
If I understand this correctly, it was failing because `sensors_3d` is None and calling `update` with None is throwing an exception?

Reply:
Based on the error message - I think it was actually that it added an empty dictionary inside the list of parameters (although I may be misreading the somewhat cryptic error)

Reply:
Ah - ok - the sensors_3d.yaml file EXISTS, but is empty..